python/tach/check.py

```diff
@@ -149,11 +150,22 @@ def check(
             if nearest_module is None:
                 continue
 
-            project_imports = get_project_imports(
-                ".",
-                file_path,
-                ignore_type_checking_imports=project_config.ignore_type_checking_imports,
-            )
+            try:
+                project_imports = get_project_imports(
+                    ".",
+                    file_path,
+                    ignore_type_checking_imports=project_config.ignore_type_checking_imports,
+                )
+            except SyntaxError:
+                print(
+                    f"{BCOLORS.WARNING}Skipping file '{file_path}' due to a syntax error.{BCOLORS.ENDC}"
+                )
+                continue
+            except OSError:
+                print(
+                    f"{BCOLORS.WARNING}Skipping file '{file_path}' due to a file system error.{BCOLORS.ENDC}"
+                )
```

Comment:
```suggestion
                    f"{BCOLORS.WARNING}Skipping '{file_path}' due to a syntax error.{BCOLORS.ENDC}"
                )
                continue
            except OSError:
                print(
                    f"{BCOLORS.WARNING}Skipping '{file_path}' due to a file system error.{BCOLORS.ENDC}"
                )
```
small nit, `file` is prob self explanatory for a file path